doc/_admin-guide/060_Sources/153_stats_exporter/000_stats_exporter_source_options.md

```diff
@@ -2,10 +2,10 @@
 title: stats-exporter() source options
 id: adm-src--stats-exporter-opt
 description: >-
-    The stats-exporter() sources directly can serve the output of syslog-ng-ctl stats and syslog-ng-ctl query to a HTTP Scraper. Unlike the stats-exporter-dont-log() that suppresses log messages from incoming scraper requests, the stats-exporter() logs the unparsed messages, storing incoming scraper HTTP requests in the MSG field.
+    The `stats-exporter()` sources directly can serve the output of `syslog-ng-ctl` stats and `syslog-ng-ctl` query to an HTTP Scraper. Unlike the stats-exporter-dont-log() that suppresses log messages from incoming scraper requests, the `stats-exporter()` logs the unparsed messages, storing incoming scraper HTTP requests in the `MSG` field.
```

Comment:
these quotes (and all of the added ones below) will remove the auto-links

https://syslog-ng.github.io/admin-guide/060_Sources/153_stats_exporter/000_stats_exporter_source_options